Add download modal to front page

Soitora opened the pull request:
## Changes
- Add a modal ([avil13/vue-sweetalert2](https://github.com/avil13/vue-sweetalert2)) for downloading the different versions instead of having multiple buttons.
- Changes the hero image with the simplified app logo.
- Replace the bottom `screens.png` image with individual feature images that can be changed using the index README

Soitora commented:
Download modal links to a non-existant FAQ entry, but I'll add that later today, feels out-of-scope for this PR.

arkon commented on src/.vuepress/styles/index.styl:
```diff
@@ -104,21 +104,52 @@ user-select()
+		a
+			border-radius 0.25rem
+			cursor pointer
+			font-family 'Open Sans', Roboto, -apple-system, BlinkMacSystemFont, Segoe UI, Oxygen, Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif
```
We should just have a variable or something for this somewhere.

arkon commented on src/.vuepress/theme/components/Home.vue:
```diff
@@ -17,9 +17,12 @@
+					@click="showDownloads"
+					class="action-button download"
+				>
+					Download ↓
```
Is it possible to use a Material icon here?

arkon commented on src/.vuepress/theme/components/Home.vue:
```diff
@@ -80,6 +87,71 @@ export default {
+				confirmButtonAriaLabel: 'Stable',
+				cancelButtonText: 'Dev',
+				cancelButtonAriaLabel: 'Dev',
+				footer: '<a href="/help/faq/application/#what-is-the-dev-version-of-tachiyomi">Which version do I select?</a>',
```
Maybe we can point to https://deploy-preview-113--tachiyomi.netlify.com/help/guides/forks/ and update that to also describe stable/dev?

(And maybe rename it to something else)

Soitora replied:
Sounds good, I'll remove the footer for now and reinclude it when the updated page is there.
Could be called "Editions" or "Versions" instead of "Forks"?

Soitora commented:
> Is there any way to make it less... bouncy?

Changed it to a fade-in instead using [daneden/animate.css](https://github.com/daneden/animate.css).
Also changed according to your other comments

I hid the footer until we fix Forks to maybe be called Editions or a synonym of that instead

Soitora commented:
On load-in the Material Icon causes something very weird to happen, and I really don't know how to solve it.
![image](https://user-images.githubusercontent.com/10836780/74088114-06072180-4a93-11ea-90c8-7e99bf33b368.png)

arkon commented:
I think in that case, it's fine to keep it as it was before (with the arrow text).

Soitora commented:
I reverted back to the symbol

I did find a promising [Material Design library](https://github.com/robcresswell/vue-material-design-icons)(?), it doesn't do `<i>` and `<div>` tags, it renders them purely as `svg`. I'll check back on it but not in this PR.

Soitora commented:
Also addresses #44
